### ISSUE TITLE
End of support message reappears when I log in to GitHub

### ISSUE THREAD

**Issue by ByRodrigoSoares** [Feodor2/Mypal#196]
**In the next release will this be resolved ??**

**Comment by Feodor2** [comment]
29 may be, but they do not like us who do not obey them and do not use chrome, thats why i am looking new place instead github

**Comment by ByRodrigoSoares** [comment]
you are absolutely right they always hinder the functioning of browsers that are not chrome thanks for answering we know of your efforts to continue supporting us
